src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,18 @@
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+
+  max-width: getWidth($-amount-of-cards, $-width-card, $-card-hor-margin);
```

Comment:
Are you sure that when you use max-width, you have 4 cards left when you reduce the screen width?